app/models/bus_stop.rb

```diff
@@ -210,20 +91,9 @@ def self.to_csv(limited_attributes: false)
     attrs = if limited_attributes
               LIMITED_ATTRS
             else
-              hashed_columns = columns.to_h { |c| [c.name, c.name.humanize] }
-                                      .except('name',
-                                              'hastus_id',
-                                              'id',
-                                              'updated_at',
-                                              'created_at',
-                                              'route_list',
-                                              'completed_at',
-                                              'completed_by',
-                                              'completed')
-              LIMITED_ATTRS.merge hashed_columns
+              LIMITED_ATTRS.merge columns.to_h { |c| [c.name.to_sym, c.name.humanize] }.except(LIMITED_ATTRS.keys)
```

Comment:
The `except` shouldn't actually matter here except that I'm pretty sure the order of the hash keys matters here. Sure, that's not the greatest idea, but...